codecov.yml

```diff
@@ -1,3 +1,3 @@
 # CHANGE THIS
 fixes:
-   - "/ansible_collections/NAMESPACE/COLLECTION/::"
+   - "/lowlydba/lowlydba/sqlserver/::"
```

Comment:
This will need to use the path format as shown in the source, starting with `ansible_collections/`